notebooks/en/medical_question_answering.ipynb

```diff
@@ -0,0 +1,5970 @@
+{
```

Comment:
<p>I think it's easier to read if you give the model one question at a time. The output is a bit harder to read since there the user has to scroll through so much text.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/272/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>